pkg/values/value_test.go

```diff
@@ -23,6 +24,13 @@ func Test_Value(t *testing.T) {
 				mv, err := NewMap(m)
 				return m, mv, err
 			},
+			mustNewValue: func() (any, Value) {
+				m := map[string]any{
+					"hello": "world",
+				}
+				mv := MustNewMap(m)
```

Comment:
Did you consider declaring one generic func?
```suggestion
				mv := must(NewMap(m))
```
```go
func must[T any](t T, err error) T {
	if err != nil { panic(err) }
	return t
}
```
And if the usages are only from this package, we should prefer not to export it.